HACKING.adoc

```diff
@@ -229,6 +229,74 @@ opam switch create my-switch-name --empty
 opam pin add ocaml-variants.$VERSION+branch git+https://$REPO#branch
 ----
 
+=== Incremental builds with `opam`
+
+If you want to fix a bug that needs opam packages to reproduce, or want to try
+a new feature that you're still actively developing on opam packages, you will
+likely need a faster feedback loop than re-creating a new switch or recompiling
+every package each time you make a small change to your code.
+
+There are two mostly orthogonal steps you can take to accelerate this:
+- Use your locally built copy, and only run the install step
+- Prevent your changes from automatically triggering the rebuild of all packages
+
+For the first step, one solution is to use the `--assume-built` flag to
```

Comment:
Since the section felt as self-contained it is unclear that you are talking about the *pinned* compiler at that point.